hwdb.d/60-evdev.hwdb

```diff
@@ -786,3 +786,51 @@ evdev:input:b0003v172Fp0047*
 evdev:input:b0003v172Fp0505*
  EVDEV_ABS_00=::160
  EVDEV_ABS_01=::160
+
+#########################################
+# Granite Devices Simucube wheel bases
```

Comment:
these are loosely sorted by vendor name, so can you move this new section up?

Reply:
Sure, I moved it up in between Google and HP.